wled00/FX_fcn.cpp

```diff
@@ -1701,6 +1700,13 @@ bool WS2812FX::deserializeMap(uint8_t n) {
   return true;
 }
 
+uint16_t WS2812FX::getMappedPixelIndex(uint16_t index) {
+  if ((realtimeMode == REALTIME_MODE_INACTIVE || realtimeRespectLedMaps)
+    && index < customMappingSize) index = strip.customMappingTable[index]; // convert logical address to physical
```

Comment:
Why `strip.`? You are in `WS2812FX`class.
I'd also reverse the order of &&.

Reply:
`strip.` was a c/p leftover, sry!